its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/ScannerMSBuildTest.java

```diff
@@ -1487,7 +1491,7 @@ private BuildResult runNetCoreBeginBuildAndEnd(Path projectDir, ScannerClassifie
   private BuildResult runBeginBuildAndEndForStandardProject(Path projectDir, String projectName, Boolean setProjectBaseDirExplicitly, Boolean useNuGet) {
     String token = TestUtils.getNewToken(ORCHESTRATOR);
     String folderName = projectDir.getFileName().toString();
-    ScannerForMSBuild scanner = TestUtils.newScanner(ORCHESTRATOR, projectDir, token)
+    ScannerCommand scanner = TestUtils.newScannerBegin(ORCHESTRATOR, folderName, projectDir, token, ScannerClassifier.NET_FRAMEWORK)
       .addArgument("begin")
```

Comment:
Why do we need to do `.addArgument("begin")` when the command is called `TestUtils.newScannerBegin`? This looks redundant to me.

Reply:
It is redundant and I will remove it in upcoming PRs. For now, I did not want to pollute this PR with way too many changes.